### DISCUSSION TITLE
Installation fails due to data quoata

### DISCUSSION THREAD

**Discussion by bjp97** [PatBall1/detectree2#134] [General]
I tried to install detectree2 in Google Colab.
I got the following error message:
This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
Until yesterday, installation worked without issues.

**Comment by PatBall1** [comment]
@bjp97 thanks for alerting me to this. I've added some more bandwidth. It should be working now.

**Reply by bjp97** [reply]
Thank you for solving the issue. It now is working again.
